lib/apiUrl.js

```diff
@@ -5,11 +5,15 @@
 		qs = require('qs'),
 		protocol = 'https',
 		host = process.env.PIPEDRIVE_API_HOST || 'api.pipedrive.com',
+		proxyHost = process.env.PIPEDRIVE_API_HOST || 'api-proxy.pipedrive.com',
```

Comment:
api-proxy is going to be deprecated, regular `api.pipedrive.com` works now as well afaik

Reply:
@tot-ra You are correct.